src/main/java/mate/academy/ishop/lib/Injector.java

```diff
@@ -25,11 +26,14 @@
     public static void injectDependencies() throws IllegalAccessException {
         for (Class certainClass : classList) {
             for (Field field : certainClass.getDeclaredFields()) {
-                if(field.getDeclaredAnnotation(Inject.class) != null){
+                if (field.getDeclaredAnnotation(Inject.class) != null) {
                     Object implementation = AnnotatedClassMap
                             .getImplementation(field.getType());
-                    if(implementation.getClass().getDeclaredAnnotation(Dao.class) != null ||
-                            implementation.getClass().getDeclaredAnnotation(Service.class) != null){
+                    if (implementation.getClass()
+                            .getDeclaredAnnotation(Dao.class) != null
+                            ||
+                            implementation
```

Comment:
@Sasha192 
```suggestion
                          ||  implementation
```
краще отак зроби, не прикольно коли || просто висить в повітрі